app/controllers/concerns/authentication.rb

```diff
@@ -14,7 +14,7 @@ def authenticated?
   def authenticate
     unless authenticated?
       session["post_dfe_sign_in_path"] = request.fullpath
-      if AuthenticationService.mode == :persona
+      if AuthenticationService.mode == "persona"
```

Comment:
Could we do `if AuthenticationService.persona?` instead?

Reply:
Done